src/coinbase/address/developer_address.ts

```diff
@@ -0,0 +1,364 @@
+import { Decimal } from "decimal.js";
+import { ethers } from "ethers";
+import { Address } from "../address";
+import { Address as AddressModel } from "../../client";
+import { Asset } from "../asset";
+import { Balance } from "../balance";
+import { BalanceMap } from "../balance_map";
+import { Coinbase } from "../coinbase";
+import { ArgumentError, InternalError } from "../errors";
+import { FaucetTransaction } from "../faucet_transaction";
+import { Trade } from "../trade";
+import { Transfer } from "../transfer";
+import { Amount, Destination, TransferStatus } from "../types";
+import { delay } from "../utils";
+import { Wallet as WalletClass } from "../wallet";
+
+/**
+ * A representation of a blockchain address, which is a developer-controlled account on a network.
+ */
+export class DeveloperAddress extends Address {
+  private _model: AddressModel;
+  private _key?: ethers.SigningKey;
+
+  /**
+   * Initializes a new Developer Address instance.
+   *
+   * @param model - The address model data.
+   * @param key - The ethers.js SigningKey the Address uses to sign data.
+   * @throws {InternalError} If the address model is empty.
+   */
+  constructor(model: AddressModel, key?: ethers.SigningKey) {
+    if (!model) {
+      throw new InternalError("Address model cannot be empty");
+    }
+    super(model.network_id, model.address_id);
+
+    this._model = model;
+    this._key = key;
+  }
+
+  /**
+   * Returns a string representation of the developer address.
+   *
+   * @returns {string} A string representing the developer address.
+   */
+  public toString(): string {
+    return `Coinbase:DeveloperAddress{addressId: '${this.getId()}', networkId: '${this.getNetworkId()}', walletId: '${this.getWalletId()}'}`;
+  }
+
+  /**
+   * Returns the wallet ID.
+   *
+   * @returns {string} The wallet ID.
+   */
+  public getWalletId(): string {
+    return this._model.wallet_id;
+  }
+
+  /**
+   * Sets the private key.
+   *
+   * @param key - The ethers.js SigningKey the Address uses to sign data.
+   * @throws {InternalError} If the private key is already set.
+   */
+  public getKey(key: ethers.SigningKey) {
+    if (this._key !== undefined) {
+      throw new InternalError("Private key is already set");
+    }
+    this._key = key;
+  }
+
+  /**
+   * Returns the list of balances for the address.
+   *
+   * @returns {BalanceMap} - The map from asset ID to balance.
+   */
+  public async listBalances(): Promise<BalanceMap> {
+    const response = await Coinbase.apiClients.address!.listAddressBalances(
+      this._model.wallet_id,
+      this._model.address_id,
+    );
+
+    return BalanceMap.fromBalances(response.data.data);
+  }
+
```

Comment:
`export` method only exist on `WalletAdress` class

Reply:
Can you clarify a bit more what this means? export on line 83?